### PR TITLE
Config UI translation for DE

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/jobs/jobs_config.js [hunk 1]
```diff
@@ -16,8 +17,9 @@ UserConfig.registerOptions('jobs', {
       id: 'LowerOpacityOutOfCombat',
       name: {
         en: 'Lower ui opacity when out of combat',
+        de: 'Veringere die Deckkraft auserhalb des Kampfes',
         fr: 'Diminiuer l\'opacité de l\'UI si hors combat',
-        cn: '非战斗状态时使UI透明化',
+        cn: '仅显示团辅监控',
```
Comment (quisquous) [inline ui/jobs/jobs_config.js]: Did you mean to edit this chinese text?

Comment (Akurosia) [reply ui/jobs/jobs_config.js]: not really tbh and honnestly dont know where this comes from o.o

Comment (Akurosia) [reply ui/jobs/jobs_config.js]: not really tbh and honnestly dont know where this comes from o.o but its now reverted

**Comment by quisquous** [comment]
Thanks! :+1:

**Pull request merged**